wolfssl/ssl.h

```diff
@@ -674,6 +674,21 @@ enum AlertLevel {
     alert_fatal   = 2
 };
 
+/* RETURN_CODE macro
+ * Some OpenSSL APIs specify "0" as the return value when an error occurs.
+ * However, some corresponding wolfSSL APIs(eg.
+ * wolfSSL_CTX_load_verify_locations) return negative values. Such functions
```

Comment:
We need to document the exact APIs that are currently supported.  Not a subset as an example.

Reply:
Added comment about the usage and about exact APIs that use this macro.